microshift_configuring/microshift-things-to-know.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="microshift-things-to-know"]
+= About responsive restarts and security certificates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-configuring
+toc::[]
+
+{product-title} responds to system configuration changes and restarts after alterations are detected, including IP address changes, clock adjustments, and security certificate age.
+
+[id="microshift-ip-address-clock-changes_{context}"]
+== IP address changes or clock adjustments
+{product-title} depends on device IP addresses and system-wide clock settings to remain consistent during its runtime. However, these settings may occasionally change on edge devices, such as DHCP or Network Time Protocol (NTP) updates.
+
+When such changes occur, some {product-title} components may stop functioning properly. To mitigate this situation, {product-title} monitors the IP address and system time and restarts if either setting change is detected.
+
+The threshold for clock changes is if the time is adjusted by greater than 10 seconds in either direction. Smaller drifts on regular time adjustments performed by the Network Time Protocol (NTP) service do not cause a restart.
```

Comment:
I think here`is` is not required
`{product-title} responds to system configuration changes and restarts after alterations are detected, including IP address changes, clock adjustments, and security certificate age.`